src/main/java/io/stargate/sgv2/jsonapi/metrics/JsonApiMetricsConfig.java

```diff
@@ -1,4 +1,4 @@
-package io.stargate.sgv2.jsonapi.api.v1.metrics;
+package io.stargate.sgv2.jsonapi.metrics;
```

Comment:
for now, leave the config classes where they are , we have been resaonably good at having them in one place.